src/containers/RepoListContainer.ts

```diff
@@ -0,0 +1,39 @@
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+
+import actions from "../actions";
+import { AppRepoList } from "../components/AppRepoList";
+import { AppRepository } from "../shared/AppRepository";
+import { IStoreState } from "../shared/types";
+
+interface IRouteProps {
+  match: {
+    params: {
+      repo: string;
+    };
+  };
+}
+
+function mapStateToProps({ repos }: IStoreState, { match: { params } }: IRouteProps) {
+  return {
+    repos: repos.repos,
+  };
+}
+
+function mapDispatchToProps(dispatch: Dispatch<IStoreState>) {
+  return {
+    deleteRepo: async (name: string, namespace: string = "kubeapps") => {
```

Comment:
I think this would be better if we kept these as actions that we dispatch and have the actual logic in the actions. Also, that way the `deleteRepo` action could call `fetchRepos` after deleting the repo.